recipes/qt/6.x.x/conanfile.py

```diff
@@ -516,8 +519,10 @@ def generate(self):
             tc.variables["HAVE_openssl"] = "ON"
             if self.options["openssl"].shared:
                 tc.variables["INPUT_openssl"] = "runtime"
+                tc.variables["QT_FEATURE_openssl_runtime"] = "ON"
             else:
                 tc.variables["INPUT_openssl"] = "linked"
+                tc.variables["QT_FEATURE_openssl_linked"] = "ON"
 
```

Comment:
```suggestion

        if is_msvc(self):
            tc.cache_variables["CMAKE_TRY_COMPILE_CONFIGURATION"] = str(self.settings.build_type)
```
workaround from https://github.com/conan-io/conan/pull/12600#issuecomment-1375663736